client/gui/console.cpp

```diff
@@ -221,7 +222,7 @@ namespace tec {
 	}
 
 	void ConsoleSink::log(const spdlog::details::log_msg& msg) {
-		std::string str(msg.payload.data());
+		std::string str(msg.payload.data(), msg.payload.size());
```

Comment:
Nice optimization 👍🏻

Reply:
this is actually a bug fix to prevent it from spewing garbage into the console

Reply:
Oh I tried to fix that one day, glad you figured it out.